src/dialogs/examples/basic/PopupSignInDialog.ts

```diff
@@ -0,0 +1,51 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import * as config from "config";
+
+export class PopupSignInDialog extends TriggerActionDialog {
+
+    private static async PopUpSignIn(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
+        let cards = new Array<builder.HeroCard>();
+
+        let buttons = new Array<builder.CardAction>();
+            /**
+             *  This is PopUp SignIn Dialog Class.
+             *  main purpose of this class is to Display the PopUp SignIn Card
```

Comment:
1. Should this be here, or up on line 6, as a comment to the class itself?
2. I'd say something like this instead "Demonstrates using a signin action to show a login page in a popup"

Reply:
Done.